raiden/network/transport/matrix.py

```diff
@@ -811,15 +811,15 @@ def _get_unlisted_room(self, room_name, invitees: List[User]):
                 room = self._client.join_room(room_name_full)
             except MatrixRequestError as error:
                 if error.code == 404:
-                    self.log.info(
+                    self.log.debug(
                         f'Room {room_name_full} not found, trying to create it.',
                         error=error,
                     )
                 else:
-                    self.log.info(f'Error joining room {room_name}: '
-                                  f'{error.content} {error.code}')
+                    self.log.debug(f'Error joining room {room_name}: '
```

Comment:
since you touch this can you make it follow our style:

self.log.debug(
    f'Bla bla'
    f'More bla bla'
)

same in the one occurrence below.

Reply:
Or better: instead of `f{}`, use `structlog` kwargs for anything variable (i.e. anything but the constant first parameter aka. event)

Reply:
@andrevmatos what are the values for `error.content` and `error.code`?

Reply:
I think `str` and `int`, but I'm not sure.. just passing it to logger kwarg will work regardless of the types or values